packages/stable/src/__tests__/api/functions.unit.spec.ts

```diff
@@ -0,0 +1,151 @@
+import nock from 'nock';
+import { beforeEach, describe, expect, test } from 'vitest';
+import { mockBaseUrl, setupMockableClient } from '../testUtils';
+
+import type {
+  Function as CogniteFunction,
+  FunctionListResponse,
+  FunctionsActivationResponse,
+  FunctionsLimitsResponse,
+} from '../../api/functions/types.gen';
+
+describe('FunctionsAPI', () => {
+  let client: CogniteClient;
+  const mockFunction = {
+    id: 1,
+    createdTime: 123455234,
+    status: 'Queued',
+    name: 'myfunction',
+    externalId: 'my.known.id',
+    fileId: 1,
+    owner: 'user@cognite.com',
+    description: 'My fantastic function with advanced ML',
+    metadata: {
+      property1: 'string',
+      property2: 'string',
+    },
+    secrets: {
+      MySecret: '***',
+    },
+    functionPath: 'myfunction/handler.py',
+    envVars: {
+      MyKey: 'MyValue',
+    },
+    cpu: 1,
+    memory: 1.5,
+    runtime: 'py311',
+    runtimeVersion: 'Python 3.8.13',
+    error: {
+      code: 400,
+      message: 'Could not build function.',
+    },
+  } as CogniteFunction;
```

Comment:
Why `as CogniteFunction` vs `const mockFunction: CogniteFunction = {}`?

Reply:
Isn't doing the same?